### ISSUE TITLE
IsTaggable::TagList.delimiter = " "

### ISSUE THREAD

**Issue by baldwindavid** [jamesgolick/is_taggable#2]
Apologies if this is obvious, but where should this be placed?...
IsTaggable::TagList.delimiter = " "

My assumption was that it would be in an initializer, but that must be incorrect.

**Comment by jamesgolick** [comment]
An initializer seems fine to me.

**Comment by baldwindavid** [comment]
Hmm...I must be doing something wrong.
I have is_taggable installed as a gem and the gem config in `environment.rb` (Rails 2.3.2)...
    config.gem "giraffesoft-is_taggable", :lib => "is_taggable"

An initializer `/config/initializers/is_taggable.rb` containing...
    IsTaggable::TagList.delimiter = " "

I'll keep messing with it.  I'm not actually sure I want the " " delimiter anyway.

**Comment by baldwindavid** [comment]
I see.  The custom delimiter is only valid for setting tags, but not for retrieval.

**Comment by baldwindavid** [comment]
How about changing the `to_s` method in `is_taggable.rb` to...

```
def to_s
  join(delimiter)
end
```

**Comment by jamesgolick** [comment]
I'm not sure exactly what you mean. If you want to send me a patch or a failing test, that'd be helpful.

**Comment by baldwindavid** [comment]
Sorry.  I'll try to clarify.

I set the delimiter to a blank space in an initializer...
    IsTaggable::TagList.delimiter = " "

I then declare is_taggable on my User model...
    class User < ActiveRecord::Base
      is_taggable :tags
    end

In my form I have a text field for the tags...
    = f.text_field :tag_list

Now when I enter the tags to create the record I can enter into the text field without a comma...
    rails giraffesoft

However, when I then view the record the :tag_list text_field is pre-filled with...
    rails, giraffesoft

rather than....
    rails giraffesoft

Basically the tag_list getter is set to ", " rather than pulling in the delimiter set with `IsTaggable::TagList.delimiter = " "`

Does that make sense?

**Comment by baldwindavid** [comment]
I'll send a patch a bit later too unless this is a behavior of the plugin that is intended.

**Comment by jamesgolick** [comment]
yeah, that sounds like a bug to me. i'd love a patch

**Comment by baldwindavid** [comment]
10-4
